pkgs/applications/editors/emacs/elisp-packages/manual-packages/youtube-dl/default.nix

```diff
@@ -15,8 +15,6 @@ trivialBuild {
     sha256 = "sha256-Etl95rcoRACDPjcTPQqYK2L+w8OZbOrTrRT0JadMdH4=";
   };
 
-  buildInputs = [ emacs ];
-
   meta = with lib; {
```

Comment:
Well, since we are treewide cleaning up, removing `with lib` can be done in that step, too.

Reply:
Removing `with lib` needs more work and can be done later.

Let's focus on `version` and `buildInputs` first.

Reply:
ok.